libdistopia/src/distopia.cpp

```diff
@@ -143,7 +162,27 @@ namespace distopia {
                 vx = vmin[0];
                 vy = vmin[1];
                 vz = vmin[2];
-            };
+            }
+
+            void MinimalVectors(const V &ix, const V &iy, const V &iz,
```

Comment:
In our distance calculation function, we have a pattern of first creating the vectors, then asking the Box to minimise them.  This means that for `Distance` we need to specialise for triclinic to first place coordinates inside the primary unit cell before creating the vectors then minimising.

If we instead use this pattern, where we ask the Box to provide minimal vectors based on two sets of x, y, z coordinates, then we don't have to specialise for triclinic cases